src/bots/joss.rs

```diff
@@ -0,0 +1,30 @@
+use rand::Rng;
+
+use super::{Bot, Dilemma, Turn};
+
+#[derive(Debug, Default)]
+pub struct Joss {}
+
+impl Bot for Joss {
+    fn new() -> Self {
+        Joss {}
+    }
+
+    fn turn(&mut self, history: &[Turn]) -> Dilemma {
+        let mut rng = rand::thread_rng();
+        match rng.gen_range(0..10) {
+            // 10% of the time, betray
+            0i32 => Dilemma::Betray,
```

Comment:
I meant to mention to move the type the range:

```suggestion
        match rng.gen_range(0i32..10) {
            // 10% of the time, betray
            0 => Dilemma::Betray,
```